sources/packages/backend/apps/workers/src/controllers/supporting-user/_test_/e2e/supporting-user.controller.createIdentifiableSupportingUsers.e2e-spec.ts

```diff
@@ -33,145 +39,187 @@ describe("SupportingUserController(e2e)-createIdentifiableSupportingUsers", () =
+              },
+            ],
+          } as ApplicationData,
+        },
       );
+      const fakePayload = createFakeCreateIdentifiableSupportingUsersPayload({
+        applicationId: savedApplication.id,
+        supportingUserType: SupportingUserType.Parent,
+        parent: 2,
+        isAbleToReport: false,
+      });
 
-    // Assert
-    // Validate DB creation.
-    const updatedApplication = await db.application.findOne({
-      select: {
-        id: true,
-        supportingUsers: {
+      // Act
+      const result =
+        await supportingUserController.createIdentifiableSupportingUsers(
+          createFakeWorkerJob<
+            CreateIdentifiableSupportingUsersJobInDTO,
+            ICustomHeaders,
+            CreateIdentifiableSupportingUsersJobOutDTO
+          >(fakePayload),
+        );
+
+      // Assert
+      // Validate DB creation.
+      const updatedApplication = await db.application.findOne({
+        select: {
           id: true,
-          supportingUserType: true,
-          isAbleToReport: true,
-          fullName: true,
+          supportingUsers: {
+            id: true,
+            supportingUserType: true,
+            isAbleToReport: true,
+            fullName: true,
+          },
         },
-      },
-      relations: { supportingUsers: true },
-      where: {
-        id: savedApplication.id,
-      },
-    });
-    expect(updatedApplication.supportingUsers).toHaveLength(1);
-    const [parent] = updatedApplication.supportingUsers;
-    expect(parent).toEqual({
-      id: expect.any(Number),
-      supportingUserType: SupportingUserType.Parent,
-      isAbleToReport: false,
-      fullName: parentFullName2,
-    });
-    // Validate job result.
-    expect(result).toEqual({
-      resultType: MockedZeebeJobResult.Complete,
-      outputVariables: {
-        createdSupportingUserId: parent.id,
-      },
-    });
-  });
+        relations: { supportingUsers: true },
+        where: {
+          id: savedApplication.id,
+        },
+      });
+      expect(updatedApplication.supportingUsers).toHaveLength(1);
+      const [parent] = updatedApplication.supportingUsers;
+      expect(parent).toEqual({
+        id: expect.any(Number),
+        supportingUserType: SupportingUserType.Parent,
+        isAbleToReport: false,
+        fullName: parentFullName2,
+      });
+      // Validate job result.
+      expect(result).toEqual({
+        resultType: MockedZeebeJobResult.Complete,
+        outputVariables: {
+          createdSupportingUserId: parent.id,
+        },
+      });
+      // Validate the creation of notification for the parent declare information
+      // to be provided by the student.
+      const notificationMessageType =
+        NotificationMessageType.ParentDeclarationRequiredParentCannotReportNotification;
+      await notificationLookupAndAssertion(
+        savedApplication,
+        parentFullName2,
+        notificationMessageType,
+      );
+    },
```

Comment:
We can directly use the notificationMessageType enum similar to previous test. 
<img width="634" height="156" alt="image" src="https://github.com/user-attachments/assets/7c5983a6-c0c4-4654-b7b4-c016c0ddadd1" />